test.py

```diff
@@ -0,0 +1,29 @@
+from scipy.stats import multivariate_normal, norm
+from functools import partial
+import numpy as np
+import bayesian_changepoint_detection.online_changepoint_detection as online
+
+
+def test_multivariate():
```

Comment:
This test data needs to have multiple breakpoints. Having only one hid the flaw in my implementation, which is that it wasn't updating parameters except for `r=0 and r=t`.